build_scripts/debian/Dockerfile

```diff
@@ -16,11 +16,12 @@ RUN apt-get -y install python3-all python3-pip python3-setuptools nano git sudo
 RUN python3 -m pip install --upgrade pip
 
 # Build .deb
-RUN mkdir Repos
-RUN mkdir Repos/mssql-cli
-ADD . Repos/mssql-cli
-WORKDIR Repos/mssql-cli
+RUN mkdir /Repos
```

Comment:
So we're hardcoding and assuming a /Repos/mssql-cli? Seems fine for docker, but we can't make this assumption for dev boxes.

Reply:
For sure, I've accounted for this.